components/invitation_modal/invitation_modal_guest_step/invitation_modal_guests_step.tsx

```diff
@@ -56,33 +65,33 @@ class InvitationModalGuestsStep extends React.PureComponent {
         };
     }
 
-    onUsersEmailsChange = (usersAndEmails) => {
+    onUsersEmailsChange = (usersAndEmails: string[]) => {
         this.setState({usersAndEmails});
-        this.props.onEdit(usersAndEmails.length > 0 || this.state.channels.length > 0 || this.state.customMessage !== '' || this.state.usersInputValue || this.state.channelsInputValue);
+        this.props.onEdit(usersAndEmails.length > 0 || this.state.channels.length > 0 || this.state.customMessage !== '' || this.state.usersInputValue.length !== 0);
```

Comment:
Was it intentional to remove the test on `channelsInputValue`?

Reply:
Ops it was a mistake